steel/src/host/mod.rs

```diff
@@ -31,6 +31,10 @@ use revm::primitives::HashMap;
 pub mod db;
 pub mod provider;
 
+/// Alias for readability, do not make public.
+#[cfg(feature = "host")]
```

Comment:
The entire `host` module is behind that flag, so I don't think this is actually necessary.

Reply:
you're right, that was a sloppy refactor :D